package-manifest.toml

```diff
@@ -252,8 +252,8 @@ only_for_targets.image_type = "standard"
 # 3. Use source.type = "manual" instead of "prebuilt"
 source.type = "prebuilt"
 source.repo = "dendrite"
-source.commit = "67a45ff847989e8b8be37c099f70a7451b277c0d"
-source.sha256 = "a2d7bda00d53e1fc2f569ef9afbef4a56a82636e2d613b7668fba7b767805474"
+source.commit = "8ba0f2cde2577219f58eb768850705e9ef7249a0"
+source.sha256 = "5ca05573a21612a8f92af428200342b63b6fb6e1b6cce89a8008341cf08a4dbe"
```

Comment:
https://buildomat.eng.oxide.computer/public/file/oxidecomputer/dendrite/image/8ba0f2cde2577219f58eb768850705e9ef7249a0/dendrite-stub.sha256.txt says this should be:

05a47e1a069a376b29c9171abd8b558b4498bd2b1d623c889c6b712334d23815